geth/common/types.go

```diff
@@ -179,23 +179,23 @@ type JailCell interface {
 	Set(string, interface{}) error
 	Get(string) (otto.Value, error)
 	Run(string) (otto.Value, error)
+	Call(item string, this interface{}, args ...interface{}) (otto.Value, error)
```

Comment:
Let make `this` -> `binding` or `context`.

Reply:
@influx6 similar comment at the top of this PR. We mimicking `otto.Call` parameters naming, and `this` actually may refer to the value or object that in JS is referred as `this`. So it's kinda confusing in a Go space, but ok when you apply it to Go<->JS bridge context.

Reply:
Check here for reasons: https://github.com/status-im/status-go/pull/290#discussion_r136698750

Reply:
Ah i see. Ok.